psycop/common/feature_generation/sequences/cohort_definer_to_prediction_times.py

```diff
@@ -98,6 +98,7 @@ def create_prediction_times(
     patients = PatientLoader.get_split(
         event_loaders=[DiagnosisLoader()],
         split=SplitName.TRAIN,
+        min_n_visits=5,
```

Comment:
We probably want this to be an argument to the DiagnosisLoader, not the PatientLoader - at least for now :-)